app/models/auditing/UserHasPenaltyAuditModel.scala

```diff
@@ -46,6 +46,9 @@ case class UserHasPenaltyAuditModel(
       ""
   }
 
+  private val ammountOfLspChargesPaid: Int = penaltyDetails.lateSubmissionPenalty.map(_.details.count(point => point.chargeOutstandingAmount.contains(0)
```

Comment:
Also do we want to filter out any appealed penalties?

Reply:
Fixed ✅